src/patternfly/components/Dropdown/examples/Dropdown.md

```diff
@@ -173,11 +182,24 @@ The dropdown panel is provided for flexibility in allowing various content withi
 {{/dropdown}}
 ```
 
+```hbs title=Dropdown-with-avatar-and-info
+{{#> dropdown id="dropdown-with-avatar-and-info-example" dropdown--IsAvatarInfo="true" dropdown--IsExpanded="true"}}
```

Comment:
I wonder if this example should just show a basic usage of the text modifier on a dropdown item since it might be confusing and not obvious that the styles in this menu depend on the page component's css.